### PR TITLE
Assume YBR_FULL after decompression in case of YBR_FULL_422

### PR DESCRIPTION
**Pull request by kubinio123**

### CODE REVIEW

File: dicom_decompress/main.py [hunk 1]
```diff
@@ -56,6 +56,8 @@ def decompress(dataset, ts):
         dataset['PixelData'].VR = 'OW'
     if dataset.pixel_array.size > 0:
         dataset.PixelData = dataset.pixel_array.tobytes()
+    if dataset.PhotometricInterpretation == 'YBR_FULL_422':
```
Comment (karl-exini) [inline dicom_decompress/main.py:59]: I think we can make this more precise. Isn't the case we have evidence for where transfer syntax is jpeg baseline process 1 (1.2.840.10008.1.2.4.50) and photometric interpretation is YBR_FULL_42, so maybe we should restrict it to this for now?

File: dicom_decompress/main.py [hunk 2]
```diff
@@ -28,16 +28,16 @@
...
 jpeg2000_photometric_interpretations = [
     pi_ybr_rct,
     pi_ybr_ict
 ]
 
-supported_photometric_interpretations = [
+supported_photometric_interpretations = [   
     pi_palette,
-    'YBR_FULL',
-    'YBR_FULL_422'
```
Comment (karl-exini) [inline dicom_decompress/main.py:40]: Maybe keep this then if the comment below holds true.

**Review by karl-exini — COMMENTED** [review]
A couple of simple questions

File: dicom_decompress/main.py (hunk 1 above)
Comment (kubinio123) [reply dicom_decompress/main.py]: Good point, updated the code :)

**Review by karl-exini — APPROVED** [review]

**Pull request merged**